src/message/messageActionSheet.js

```diff
@@ -167,34 +172,34 @@ const deleteTopic = async ({ auth, stream, topic, dispatch, _ }) => {
       );
     });
   if (await AsyncAlert()) {
-    await dispatch(deleteMessagesForTopic(stream, topic));
+    const sub = subscriptions.find(x => x.stream_id === streamId);
```

Comment:
It should be fairly simple to rewrite `deleteMessagesForTopic` to take a stream ID — do that instead of finding the stream name here. Again, I'd do that in a commit before this one.

Reply:
yes, did the necessary changes.